cmd/thyra-server/main.go

```diff
@@ -36,6 +37,7 @@ func main() {
 
 	certFilePtr := flag.String("tls-certificate", "", "path to certificate file")
 	keyFilePtr := flag.String("tls-key", "", "path to key file")
+	massaNetworkAddressPtr := flag.String("network", "", "IP address of the chosen massa network")
```

Comment:
Somthing like nodeUrl ?

Reply:
yes indeed think nodeUrl is a better choice to avoid confusion

Reply:
Are you talking about the variable's name or the flag's name ?
If it's the latter, then I think that nodeUrl isn't necesseraly the right name because it doesn't imply straight away that it also accepts an IP address. WDYT ?

Reply:
Yeah but as address is for SC, I personnally may be confused

Reply:
What about nodeServer or simply node ?

Reply:
yeah go for `nodeServer` :)